Bottom: d24008e27fa957552d14de5ca8fc8abccdb63eb6
Top:    2b50ccfae9781b51735375585748c6dfd78afcd8
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-22 13:57:09 -0500

Refresh of all-normalize-vars

---
diff --git a/roles/docker-svn/tasks/restore.yml b/roles/docker-svn/tasks/restore.yml
index f8bf2a2..22e7204 100644
--- a/roles/docker-svn/tasks/restore.yml
+++ b/roles/docker-svn/tasks/restore.yml
@@ -6,7 +6,7 @@
 
 - name: find files
   files_in_dir:
-    path: '{{ host_svn_docker_restore_dir }}'
+    path: '{{ svn_docker_restore_dir }}'
   register: svn_repo_backup
 
 - name: data from tape
@@ -20,12 +20,12 @@
 
 - name: find files
   files_in_dir:
-    path: '{{ host_svn_docker_restore_dir }}'
+    path: '{{ svn_docker_restore_dir }}'
   register: svn_repo_backup
 
 # - name: File permissions
 #   file:
-#     path: '{{ host_svn_docker_restore_dir }}/{{ item }}'
+#     path: '{{ svn_docker_restore_dir }}/{{ item }}'
 #     state: touch
 #     owner: root
 #     group: root
@@ -40,7 +40,7 @@
     detach: False
     volumes:
       - '{{ svn_dv_name }}:/var/lib/svn:z'
-      - '{{ host_svn_docker_restore_dir }}:/tmp/import_export:z'
+      - '{{ svn_docker_restore_dir }}:/tmp/import_export:z'
     image: '{{ svn_image_name }}:{{ docker_image_tag }}'
     command: ['import', '{{ svn_repos }}']
     name: 'svn_restore'
@@ -55,7 +55,7 @@
 
 - name: Remove files
   file:
-    path: '{{ host_svn_docker_restore_dir }}'
+    path: '{{ svn_docker_restore_dir }}'
     state: absent
 
 - name: State file
